paddle/fluid/framework/selected_rows.h

```diff
@@ -48,6 +48,11 @@ class SelectedRows {
  public:
   SelectedRows(const std::vector<int64_t>& rows, const int64_t& height)
       : rows_(rows), height_(height) {
+    for (auto row : rows_) {
```

Comment:
This enforce can be expensive?

Reply:
Leave this to the caller? feels expensive to me.

Reply:
Done